assemblies/assembly_openshift-ansible-playbook/main.adoc

```diff
@@ -32,6 +33,11 @@ parameters as necessary. A {PRODUCT} Ansible Playbook for the Automation Broker
 `ReadWriteMany` mode is not supported on OpenShift Online and OpenShift Dedicated.
 ====
 
+[NOTE]
+====
+Since {PRODUCT} version 7.5, support for {OPENSHIFT} 3.x, including installation using all templates and using the Automation Broker (Ansible Playbook) is deprecated. New features might not be added, and this functionality will be removed in a future release. 
```

Comment:
@mramendi Just this one comment. I didn't take the time to look at every rendered output, just a couple.

The current wording is a tad rough to my ear. I suggest the following or similar for all:

> Beginning with {PRODUCT} version 7.5, support for {OPENSHIFT} 3.x is deprecated, including installation using all templates and using the Automation Broker (Ansible Playbook). New features might not be added, and this functionality will be removed in a future release.